test/unit/test_filters.py

```diff
@@ -1,9 +1,18 @@
-from src.filters import remove_repeats, remove_special_characters
+from src.console_utils import filter_badwords, filter_badwords_adjacent_words
+from src.filters import remove_repeats, remove_endings
+import unittest
 
```

Comment:
I dont think we are sticking to 'src.filename' convention as we can mark src folder as a root.  @SunBear1  should confirm this

Reply:
By marking `src` directory as `Sources root` we don't have to use src prefix. Please refactor our code under this condition.